### PR TITLE
Add page for course 

### PR DESCRIPTION
**Pull request by joshbouncesecurity**

### CODE REVIEW

**Comment by joshbouncesecurity** [comment]
@avidouglen just a reminder that we are waiting to merge this!

File: valuenotvulns.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# Building a High-Value AppSec Scanning Programme (SCA, SAST, DAST and more)
+</div>
+
+Thanks for your interest in “Building a High-Value AppSec Scanning Programme”! 
+
+You can register directly via [this link](https://appsecg.host/bhreg) or scroll down for more information about the course.
+
+Also, if you submit your email address in the form below before you register and then attend the course, we will send you an exclusive gift!
+
+<iframe scrolling="no" width="750px" height="390px" src="https://forms.office.com/Pages/ResponsePage.aspx?id=cRoiiPFjO02bvb320DZszAO9NGOU1VxFiFNicVg9yJdUNVlYUUFFNVVIRFFQT1JSSzMwNDRaNk9GQy4u&embed=true" frameborder="0" marginwidth="0" marginheight="0" style="overflow:hidden; border: none; max-width:100%; max-height:100vh" allowfullscreen webkitallowfullscreen mozallowfullscreen msallowfullscreen> </iframe>
```
Comment (avidouglen) [inline valuenotvulns.md:17]: For next time, I'm sure we can sort out better solution for this....

File: valuenotvulns.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+_"Very good, josh is higly (sic) skilled in the topic."_
+
+<a id="audiovisual-information-about-the-course1" class="anchor"></a>
+### Audio/Visual information about the course ([↑](#contents))
+
+#### Elevator pitch for the course (~2 minutes)
+
+In this short video, I give a quick explanation of the course and the ideas around it.
+
+<iframe class="itemvid" width="640" height="360" src="https://www.youtube.com/embed/_fzC-xmtwHc" title="Building a High Value AppSec Scanning programme - Black Hat EU 2022 - Josh Grossman" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (avidouglen) [inline valuenotvulns.md:87]: Is this the same video from last year? It says 2022 - or is that a typo...

**Comment by joshbouncesecurity** [comment]
My plan is to create an updated video

**Comment by joshbouncesecurity** [comment]
Like what? 😊


Josh Grossman
Bounce Security
Mobile: +972-54-816-5820

**Pull request merged**